docroot/modules/custom/mass_map/src/MapLocationFetcher.php

```diff
@@ -249,7 +249,7 @@ public function getLocations(array $nids) {
         // Get hours from the referenced contact info node.
         if (!$contact_information_entity->field_ref_hours->isEmpty()) {
           $hours_paragraph = $contact_information_entity->field_ref_hours->entity;
-          if (!is_null($hours_paragraph) &&   !is_null($hours_paragraph->field_hours_structured->entity)) {
+          if (!is_null($hours_paragraph)) {
```

Comment:
@clairesunstudio @vordude This would also fix the issue
```suggestion
          if (!is_null($hours_paragraph) && !isset($hours_paragraph->field_hours_structured->entity)) {
```